backend/corpora/lambdas/api/v1/collection.py

```diff
@@ -61,6 +61,32 @@ def get_collections_list(from_date: int = None, to_date: int = None, token_info:
     return make_response(jsonify(result), 200)
 
 
+@dbconnect
+def get_collections_curation(visibility: str, token_info: dict):
+    """
+    Collections index endpoint for Curation API. Only return Collection data for which the curator is authorized.
+    @param visibility: the CollectionVisibility in string form
+    @param token_info: access token info
+    @return: Response
+    """
+
+    collections = Collection.list_collections_for_curator_api(g.db_session, visibility)
+    allowed_collections = []
+    for collection in collections:
+        owner = collection["owner"]
+        if is_user_owner_or_allowed(token_info, owner):
+            collection["access_type"] = "WRITE"
+        elif collection["visibility"] == "PRIVATE":
+            continue
+        else:
+            collection["access_type"] = "READ"
+        del collection["owner"]  # Don't actually want to return 'owner' in response
+        collection["collection_url"] = f"https://www.cellxgene.cziscience.com/collections/{collection['id']}"
```

Comment:
This domain-only works for prod.

Reply:
Fixed... except for the case of `test`... will address that

Reply:
Addressed `test` as well